mysql-test/t/percona_slowlog_query_tables.test

```diff
@@ -0,0 +1,42 @@
+#
+# Check list of tables affected by a query is added to slow log
+#
+--source include/log_prepare.inc
+
+SET @saved_slow_query_log = @@GLOBAL.slow_query_log;
+
+SET SESSION log_slow_verbosity = 'query_info';
+SET GLOBAL slow_query_log = 1;
+SET long_query_time = 0;
+
+CREATE TABLE t1 (a INT, b INT);
+CREATE TABLE t2 (a INT, b INT);
+INSERT INTO t1 VALUES (1, 1),(2, 2),(3, 3);
+INSERT INTO t2 VALUES (1, 1),(2, 2),(3, 3);
+
+--let log_file=percona.slow_log.query_tables
+--source include/log_start.inc
+
+SELECT * FROM t1;
+
+--source include/log_stop.inc
+
+--let grep_pattern = ^# Query_tables: t1
+--let log_expected_matches = 1
+--source include/log_grep.inc
+
+--source include/log_start.inc
+
+SELECT * FROM t1, t2;
+
+--source include/log_stop.inc
+
+--let grep_pattern = ^# Query_tables: t1,t2
+--let log_expected_matches = 1
+--source include/log_grep.inc
```

Comment:
does it work, if the query has inner queries, joins etc? also CTEs

Reply:
I've added some queries into the test. Please pay attention to CTE result, tables list includes CTE name as well.